### ISSUE TITLE
[BUG] forge cleric still doesnt spawn with smithing tools

### ISSUE THREAD

**Issue by wow50000** [Rotwood-Vale/Ratwood-Keep#1224]
## Describe the bug
![image](https://github.com/user-attachments/assets/46aa8090-ad20-454d-85f5-2f4098c56160)
I have no idea why they dont spawn with the backpack

## Round ID:
![image](https://github.com/user-attachments/assets/8954717e-29f5-45d1-ad6c-7f897aa4b938)

## Steps to Reproduce:
spawn as a forge cleric

**Comment by Nakalon** [comment]
They spawn, the bagpakc is just invisible!

![image](https://github.com/user-attachments/assets/030d1a8a-6a97-47eb-b95a-3daa26ca1f33)

**Comment by Nakalon** [comment]
its... here...

![image](https://github.com/user-attachments/assets/dd39ba66-d873-48c1-aeb3-70ec128d6f4a)

**Comment by Nakalon** [comment]
I have to guess this is related to:
![image](https://github.com/user-attachments/assets/10af92fe-d1bb-4ef4-b407-f3cc7023b5bc)

However, backpack is in the left not the right:

![image](https://github.com/user-attachments/assets/e186d102-afe6-4969-a84a-71569aef49d0)

I have no idea how to fix this...

**Comment by wow50000** [comment]
yeah idk it wasnt there for me, I was clicking that shit. it seems to sometimes not spawn
